CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "parisc.*")
 	set(HARDINFO_ARCH "parisc")
 elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "sparc{32,64}")
 	set(HARDINFO_ARCH "sparc")
+elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "aarch64*")
+	set(HARDINFO_ARCH "aarch64")
```

Comment:
Within the next day or two I'll test that in the packaging on the Ubuntu infra. I'll let you know.